src/domains/ecommerce-common/02_security.tf

```diff
@@ -97,6 +97,19 @@ resource "azurerm_key_vault_secret" "notifications_sender" {
   value        = "<TO UPDATE MANUALLY ON PORTAL>"
   key_vault_id = module.key_vault.id
 
+  lifecycle {
+    ignore_changes = [
+      value,
+    ]
+  }
+}
+
+resource "azurerm_key_vault_secret" "payment-gateway-service-jwt-key" {
+  count        = var.env_short == "d" || var.env_short == "u" ? 1 : 0
```

Comment:
why this count? the secret is required for all environment

Reply:
Right, solved with 43bff35 👍